versionToServerVersionMapping.json

```diff
@@ -1,6 +1,6 @@
 [
   {
     "uiVersion": "PREVIEW",
-    "serverVersion": "r1353"
+    "serverVersion": "r1354"
```

Comment:
this only has to be updated if you add changes that require a newer server version

Reply:
I changed it back to r1353